Mesh4all/m4a-firmware
# wifi-subsys: nvs + wifi integration

### Contribution description
Factory NVS values and integration with the network component

### Testing procedure
Execute:
```sh
.../wifi-subsys/test$ idf.py build -D "TEST_COMPONENTS=network" flash monitor
```
The output should ends with something like:
```sh
m4a-firmware/wifi-subsys/components/network/test/test_wifi.c:20:Wi-Fi initialize:PASS
``` 
### Issues/PRs references
It's part of #96 fixing some related issues with storage and network components
Fix #71
